### PR TITLE
clientupdate: detect when tailscale is installed without package manager

### PR DESCRIPTION
**Pull request by awly**
On linux users can install Tailscale via package managers or direct tarball downloads. Detect when Tailscale is not installed via a package manager so we can pick the correct update mechanism. Leave the tarball update function unimplemented for now (coming in next PR!).

Updates #6995
Updates #8760

### CODE REVIEW

File: clientupdate/clientupdate.go [hunk 1]
```diff
@@ -300,6 +304,14 @@ func parseSynoinfo(path string) (string, error) {
 }
 
 func (up *updater) updateDebLike() error {
+	if err := requireRoot(); err != nil {
+		return err
+	}
+	if err := exec.Command("dpkg", "--status", "tailscale").Run(); err != nil && isExitError(err) {
+		// Tailscale was not installed via apt, update via tarball download
```
Comment (noncombatant) [inline clientupdate/clientupdate.go:311]: These comments don't match what `updateLinuxBinary` does. Are you going to change `updateLinuxBinary` in a later PR?

Comment (awly) [reply clientupdate/clientupdate.go:311]: Yep, the next PR will do the tarball download from pkgs.tailcale.com and binary replacement.
I like small incremental PRs :)

Comment (awly) [reply clientupdate/clientupdate.go:311]: Follow-up PR for `updateLinuxBinary`: https://github.com/tailscale/tailscale/pull/9144

Comment (noncombatant) [reply clientupdate/clientupdate.go:311]: Agreed re: small, incremental PRs!

**Review by noncombatant — APPROVED** [review]

**Pull request merged**